spec/defines/key_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 describe 'apt::key', :type => :define do
   let(:facts) { { :lsbdistid => 'Debian' } }
-  GPG_KEY_ID = '4BD6EC30'
+  #GPG_KEY_ID = '4BD6EC30'
```

Comment:
The comments serve no purpose, just remove them. We've got Git's history powers if we ever wanted to know what this line looked like in the past.